config.json

```diff
@@ -4,6 +4,10 @@
   "blurb": "Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.",
   "gitter": "ruby",
   "solution_pattern": "[Ee]xample|\\.meta/solutions/[^/]*\\.rb",
+  "editor": {
+    "tab_width": 2,
+    "soft_tabs": true
```

Comment:
I had never heard of "soft" tabs. Is that a common term?

Reply:
Yep. 

> Hard tabs are the ASCII tab character (9, or \t in many languages) that advance the cursor to the next tab stop. Soft tabs are the use of spaces to simulate this effect for indentation.

I've seen it in many many configuration / settings, but recently I've seen the much clearer "use spaces instead of tabs".

Reply:
Ah. I like "use spaces instead of tabs" better to be honest :)

Reply:
I think we use `indent_style` instead. "soft tabs" is a pretty standard phrase though :p

Reply:
"expand tabs" is another one that seems common.